From 1f0d9eb4b28e46a43cf7ad6f77c9fe311d51643b Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Sun, 5 Aug 2012 14:32:08 +1000
Subject: [PATCH synaptics] conf: apply ClickPad softbuttons option through
 fdi file too

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 conf/11-x11-synaptics.fdi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/conf/11-x11-synaptics.fdi b/conf/11-x11-synaptics.fdi
index d487f09..fa54f33 100644
--- a/conf/11-x11-synaptics.fdi
+++ b/conf/11-x11-synaptics.fdi
@@ -30,6 +30,10 @@
 
 	For other possible options, check CONFIGURATION DETAILS in synaptics man page
         -->
+        <merge key="input.x11_options.SoftButtonAreas">50% 0 82% 0 0 0 0 0</merge>
+        <match key="info.product" contains="Apple|bcm5974">
+            <merge key="input.x11_options.SoftButtonAreas">0 0 0 0 0 0 0 0</merge>
+        </match>
     </match>
   </device>
 </deviceinfo>
-- 
1.7.10.4

